app/controllers/sessions_controller.rb

```diff
@@ -30,6 +31,10 @@ def callback
   end
 
   def destroy
+    session[ActionInterceptor.config.default_key] = nil
+    session[:registration_return_to] = nil
+    session[:client_id] = nil
```

Comment:
@Dantemss can you bless this clearing of session vars?  (there were cases where old session data was messing up some links during registration)